### PR TITLE
Feature: refactoring guest label indicator group role label indicator

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Create a LabelIndicator view and  init method with an icon, title and accessibilityIdentifier  parameters instead of  GuestLabelIndicator and GroupRoleLabelIndicator.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/Views/LabelIndicator.swift [hunk 1]
```diff
@@ -18,43 +18,45 @@
...
+final class LabelIndicator: UIView {
+    
     private let variant: ColorSchemeVariant
-    private let groupRoleIcon = UIImageView()
+    private let indicatorIcon = UIImageView()
     private let titleLabel = UILabel()
     private let containerView = UIView()
     
-    init() {
+    init(icon: StyleKitIcon, title: String, accessibilityIdentifier: String) {
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Components/Views/LabelIndicator.swift]: We can create a context for `LabelIndicator` having value like `.groupRole` and `.guest`

File: Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift [hunk 2]
```diff
@@ -219,7 +219,6 @@ final class ProfileHeaderViewController: UIViewController, Themeable {
     
     func applyColorScheme(_ variant: ColorSchemeVariant) {
         availabilityTitleViewController.availabilityTitleView?.colorSchemeVariant = variant
-        guestIndicator.colorSchemeVariant = variant
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift]: still needed for variant update?

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift]: I set in init method `self.variant = ColorScheme.default.variant`

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift]: This method would be called after color scheme is update, which may happens after `init` is called

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift]: Add `colorSchemeVariant` to the `LabelIndicator` view.

**Pull request merged**